daemon/call.c

```diff
@@ -899,11 +899,13 @@ struct packet_stream *__packet_stream_new(struct call *call) {
 	stream = uid_slice_alloc0(stream, &call->streams);
 	mutex_init(&stream->in_lock);
 	mutex_init(&stream->out_lock);
+	mutex_init(&stream->jb.lock);
 	stream->call = call;
 	atomic64_set_na(&stream->last_packet, rtpe_now.tv_sec);
 	stream->rtp_stats = g_hash_table_new_full(g_int_hash, g_int_equal, NULL, __rtp_stats_free);
 	recording_init_stream(stream);
-	stream->send_timer = send_timer_new(stream);
+	stream->send_timer = send_timer_new(stream, 0);
+	stream->buffer_timer = send_timer_new(stream, 1);
```

Comment:
These should be either different functions (send_timer_new and jitter_buffer_new or something like that), or only use a single timer object.